src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -2410,10 +2410,23 @@ private void OnAutoEvoPredictionComplete(PendingAutoEvoPrediction run)
         autoEvoPredictionRunSuccessful = true;
 
         // Set the initial value
-        totalPopulationLabel.ResetInitialValue();
-        totalPopulationLabel.Value = run.PlayerSpeciesOriginal.Population;
+        totalEnergyLabel.ResetInitialValue();
+        totalEnergyLabel.Value = run.PlayerSpeciesOriginal.Population;
 
-        totalPopulationLabel.Value = newPopulation;
+        results.GetPatchEnergyResults(run.PlayerSpeciesNew).TryGetValue(
+            Editor.CurrentPatch, out var newTotalPatchEnergyGathered);
+        var totalEnergyLabelText = totalEnergyLabel.GetNode<Label>(totalEnergyLabel.ValuePath);
+
+        if (newTotalPatchEnergyGathered != null)
+        {
+            totalEnergyLabel.Value = newTotalPatchEnergyGathered.TotalEnergyGathered;
+            if (totalEnergyLabelText != null)
+            {
+                totalEnergyLabelText.Text =
+                    @$"{StringUtils.ThreeDigitFormat(
```

Comment:
Why is the `@` symbol here? Is it required.

Reply:
I had to somehow split the line, @ is used for strings in such case if I am familiar with it enough (Correct me if I'm wrong).

Reply:
I kind of assumed that after writing the comment. I think it would be cleaner to have a temporary variable to hold the result of ThreeDigitFormat call, that way the string interpolation should then fit on a single line.

Reply:
`string formatted = StringUtils.ThreeDigitFormat(newTotalPatchEnergyGathered.TotalEnergyGathered);`
Can I leave the name `formatted` as is? it barely fits and I can fit just up to 6 characters more there.

Reply:
I'd use `var formatted` here. `formatted` is not the most descriptive variable name but as it is used in such a small scope I think it is fine to compromise on it.